### PR TITLE
Add dragonlist test

### PR DESCRIPTION
**Pull request by ikennarichard**
## Add Dragons list test
- Use jest to mock redux and api functionality
- Add test for the dragon list component

### CODE REVIEW

File: src/test/DragonsList.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        },
+        {
+          id: '2', name: 'Dragon 2', image: 'image link', description: 'Description 2', reserved: true,
+        },
+      ],
+      isLoaded: true,
+    });
+
+    useDispatch.mockReturnValue(jest.fn());
+  });
```
Comment (mahabubx7) [inline src/test/DragonsList.test.js:24]: that's really great man :partying_face:

**Review by mahabubx7 — APPROVED** [review]
## :partying_face: Congratulations :partying_face: 

giphy

You have done a great job! Thanks for this task you have completed it pretty well.

# :heavy_check_mark: Approved :heavy_check_mark: 
- [x] No linters errors.
- [x] Maintained Git flows.
- [x] All tests are passing.

:1st_place_medal: Highlights :1st_place_medal: 
- Your implemented tests are working fine.
- Nice use of the mocking feature in tests.

Nothing more to say now. It's time to merge this.

**Review by mahabubx7 — APPROVED** [review]
## :partying_face: Congratulations :partying_face: 

giphy

You have done a great job! Thanks for this task you have completed it pretty well.

# :heavy_check_mark: Approved :heavy_check_mark: 
- [x] No linters errors.
- [x] Maintained Git flows.
- [x] All tests are passing.

:1st_place_medal: Highlights :1st_place_medal: 
- Your implemented tests are working fine.
- Nice use of the mocking feature in tests.

Nothing more to say now. It's time to merge this.

File: src/test/DragonsList.test.js (hunk 1 above)
Comment (ikennarichard) [reply src/test/DragonsList.test.js:24]: Thanks @mahabubx7

**Pull request merged**